<launch>

  <node name = "data1_reader"
        type = "relay.py"
        pkg = "opust_ros">
    <param name = "config_file"
           value = "$(find opust_ros)/relay_conf/spectrometer_data1_reader.conf" />
  </node>

  <node name = "data2_reader"
        type = "relay.py"
        pkg = "opust_ros">
    <param name = "config_file"
           value = "$(find opust_ros)/relay_conf/spectrometer_data2_reader.conf" />
  </node>

</launch>
